components/resources/wallet_strings.grdp

```diff
@@ -1172,6 +1172,9 @@
   <message name="IDS_BRAVE_WALLET_SYNC_COMPLETE_MESSAGE" desc="Sync complete message on sync dialog.">Sync complete. You may now close this window.</message>
   <message name="IDS_BRAVE_WALLET_SYNC_STARTED_MESSAGE" desc="Sync started message on sync dialog.">Please don’t close this window until sync finishes.</message>
   <message name="IDS_BRAVE_WALLET_CONTINUE_USING_WALLET" desc="Continue using wallet button label on sync dialog. ">Continue using wallet in a new tab</message>
+  <message name="IDS_BRAVE_WALLET_SHIELDED_ACCOUNT_BIRTHDAY_BLOCK" desc="Shielded account birthday block input label">Shielded account birthday block</message>
+  <message name="IDS_BRAVE_WALLET_ACCOUNT_BIRTHDAY_TO_LOW" desc="Account birthday to low error">Account birthday must be greater than <ph name="BLOCK_NUMBER">$1<ex>1687104</ex></ph></message>
+  <message name="IDS_BRAVE_WALLET_ACCOUNT_BIRTHDAY_TO_HIGH" desc="Account birthday to high error">Account birthday must be less than <ph name="BLOCK_NUMBER">$1<ex>7687104</ex></ph></message>
```

Comment:
`_TOO_LOW` and `_TOO_HIGH`, as well as `too low` and `too high` in the `desc`s.

Reply:
Fixed with https://github.com/brave/brave-core/compare/1ec5fbf5d4f2f2388e23f47e6897f4e5ab106de2..2d66c727857d4fc32cf6af8de3f29a0dd2693209